src/flapjack/transport_test.go

```diff
@@ -12,4 +12,28 @@ func TestDialFails(t *testing.T) {
 	}
 }
 
-// TODO(auxesis): add test for sending and receiving Events
+func TestSendSucceeds(t *testing.T) {
+  transport, _ := Dial("localhost:6379", 9)
```

Comment:
The indenting does not conform to [Go's norms](http://golang.org/doc/effective_go.html#formatting) - need to run this through `go fmt` I think